### PR TITLE
ECO 686/683 add user update wallet

### PR DESCRIPTION
**Pull request by liorama**
Support `PATCH` for `users/` to update wallet address (and report event).

note: Swagger update on Sunday + tests for this

### CODE REVIEW

File: scripts/src/metrics.ts [hunk 1]
```diff
@@ -15,6 +15,10 @@ export function userRegister(newUser: boolean, newWallet: boolean) {
 	statsd.increment("user_register", 1, { new_user: newUser.toString(), new_wallet: newWallet.toString() });
 }
 
+export function walletAddressUpdate(newWalletAddress: string) {
+	statsd.increment("wallet_address_update_succeeded", 1, { newWalletAddress });
```
[the remark this answers is not in the document]
Comment (liorama) [reply scripts/src/metrics.ts]: That's the name Odded gave it

**Comment by liorama** [comment]
@doodyparizada https://github.com/kinecosystem/ecosystem-api/pull/26

File: scripts/src/mock_client.ts [hunk 2]
```diff
@@ -2,6 +2,8 @@ import * as expect from "expect";
 import * as jsonwebtoken from "jsonwebtoken";
 import axios from "axios";
 
+import { init as initModels } from "./models/index";
+import "./public/app";
```
Comment (doodyparizada) [inline scripts/src/mock_client.ts]: I don't think you need this

File: tests/src/services/users.spec.ts [hunk 3]
```diff
@@ -56,8 +56,9 @@ describe("api tests for /users", async () => {
 			.set("Authorization", `Bearer ${token.id}`)
 			.expect(200)
 			.expect((res: Response) => {
-				if (res.body.stats.earn_count !== 2 || res.body.stats.spend_count !== 2)
-					throw new Error("unexpeced body " + JSON.stringify(res.body))
+				if (res.body.stats.earn_count !== 2 || res.body.stats.spend_count !== 2) {
+					throw new Error("unexpeced body " + JSON.stringify(res.body));
```
Comment (doodyparizada) [inline tests/src/services/users.spec.ts]: unexpected

**Pull request merged**